Documentation/contributor-guide/dependency_management.md

```diff
@@ -125,7 +125,14 @@ execute it for 3.4.
 
 ## Golang versions
 
-The etcd project aims to maintain a development branch that is on the latest [Go version](https://go.dev/dl), ideally, this will align with the Go version in use for Kubernetes project development. For an example of how to update etcd to a new minor release of Go refer to issue <https://github.com/etcd-io/etcd/issues/16393> and the linked pull requests.
+The etcd project aims to maintain a development branch that is on the latest [Go version](https://go.dev/dl),
+ideally, this will align with the Go version in use for Kubernetes project development. For an example of how
+to update etcd to a new minor release of Go refer to issue <https://github.com/etcd-io/etcd/issues/16393> and
+the linked pull requests.
+
+For all libraries (e.g., bbolt, raft, gofail), we should always stick to the oldest supported go version
```

Comment:
Suggestion makes sense, two comments about the phrasing:

For this case should etcd client be included as a library or are you talking about subprojects?

Also for the clearer discussion about what "go supported version" means I would go further to separate between version compatibility for libraries and version used to build official binaries. I mean the `go` and `toolchain` keywords in the `go.mod` files.

Reply:
Updated, PTAL